docs/manual/layout/templates/template-data.de.md

```diff
@@ -7,8 +7,10 @@ aliases:
 weight: 50
 ---
 
-Die verfügbaren Template-Daten variieren je nach Quelle der Vorlage. In der Regel ist der vollständige 
-Datensatz im Template über die Angabe von `$this->…` erreichbar.
+Die verfügbaren Template-Daten variieren je nach Quelle der Vorlage. 
+
+
+## PHP Template
```

Comment:
I think we should still include here that you can access variables via `$this->…`.

Reply:
II wanted to keep the descriptions more general. The corresponding queries can then be found in the code examples.
??

Reply:
Changed with the last commit